duet/views/app_store_territory_source_type_report.view.lkml

```diff
@@ -0,0 +1,139 @@
+view: app_store_territory_source_type_report {
```

Comment:
Is this the right view name? Looks like it's only the name of the first table name in the join list below